File: src/main/java/io/openfuture/openmessanger/service/response/SignUpResponse.java
package io.openfuture.openmessanger.service.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class SignUpResponse {

    String message;
    Data data;

    @lombok.Data
    @NoArgsConstructor
    @AllArgsConstructor
    public static class Data{
        String email;
        String firstName;
        String lastName;
    }

}

File: src/main/kotlin/io/openfuture/openmessenger/service/response/SignUpResponse.kt
package io.openfuture.openmessenger.service.response

data class SignUpResponse(
    var message: String? = null,
    var data: Data? = null
)

data class Data(
    var email: String? = null,
    var firstName: String? = null,
    var lastName: String? = null
)